h/static/scripts/group-forms/config.ts

```diff
@@ -8,6 +8,18 @@ export type ConfigObject = {
   styles: string[];
   api: {
     createGroup: APIConfig;
+    updateGroup: APIConfig;
+  };
+  components: {
+    CreateEditGroupForm: {
```

Comment:
I'm not convinced that structuring the config around the frontend's component structure is the right way to go, as it couples them unnecessarily.

Reply:
I've changed this to:

```javascript
export type ConfigObject = {
  ...
  context: {
    group: {
      pubid: string;
      name: string;
      description: string;
      link: string;
    } | null;
  };
};
```